composer.json

```diff
@@ -23,8 +23,10 @@
         "symfony/assetic-bundle"               : "~2.6",
         "symfony/monolog-bundle"               : "~2.7",
         "symfony/swiftmailer-bundle"           : "~2.3",
-        "symfony/symfony"                      : "~2.7",
-        "twig/extensions"                      : "~1.2"
+        "symfony/symfony"                      : "~2.7.3",
+        "twig/extensions"                      : "~1.2",
+        "knplabs/knp-paginator-bundle"         : "~2.4.2",
+        "knplabs/knp-components"               : "~1.2.4"
```

Comment:
I just want to ensure `knplabs/knp-components` install with this version at least because of [this bug](https://github.com/KnpLabs/knp-components/issues/94)